src/lib/parser/parseScripts.js

```diff
@@ -0,0 +1,32 @@
+import { parseScript } from './parseScript.js';
+import Parser from './parser.js';
+const assert = console.assert;
+
+const SIZEOF_BOX = 8;
+
+const parseScripts = (arrayBuffer, i, offset = 0) => {
+  const parser = new Parser(arrayBuffer);
+  const metadata = {
+    id: i,
+    offset,
+    size: arrayBuffer.byteLength,
+  };
+
+  const chunkSize = parser.getUint16();
+
+  assert(
+    chunkSize === arrayBuffer.byteLength,
+    'Script res size flag does not match chunk size.',
+  );
+
+  // const scriptBuffer = arrayBuffer.slice(4, 4 + arrayBuffer.length);
+  parser.getUint8();
+  parser.getUint8();
```

Comment:
Not going to lie, I'm not sure what does values are. The first one is always 0 but the second one has different values depending on the script.

Initially I thought that it might be that the size was stored in two works instead of one (as [here](https://wiki.scummvm.org/index.php?title=SCUMM/Technical_Reference/Script_resources#SCUMM_V1_and_V2_Room_Entry_and_Exit_Scripts) it says 4 bytes) but that doesn't seem to be the case.